src/server/dragonfly_test.cc

```diff
@@ -109,17 +110,13 @@ TEST_F(DflyEngineTest, Sds) {
 
 class DflyRenameCommandTest : public DflyEngineTest {
  protected:
-  DflyRenameCommandTest() : DflyEngineTest() {
+  DflyRenameCommandTest() {
     // rename flushall to myflushall, flushdb command will not be able to execute
     absl::SetFlag(
         &FLAGS_rename_command,
         std::vector<std::string>({"flushall=myflushall", "flushdb=", "ping=abcdefghijklmnop"}));
   }
-
-  void TearDown() {
-    absl::SetFlag(&FLAGS_rename_command, std::vector<std::string>({""}));
-    DflyEngineTest::TearDown();
-  }
+  absl::FlagSaver saver_;
```

Comment:
replaced the `SetFlag()` with empty vector with saver because setting to empty runs into command validation issue on next test run, which makes sense because `rename_command=""` shouldn't be a valid command.

The saver resets all custom flags set in ctor body using RAII.

Reply:
//TODO (kostas)

1. Check `TearDown()` 
2. Check which commands we use explicitly and might break because of the aliases. For example how does this affect pub/sub etc

@abhijat will follow up on this and ping.

Reply:
for (2) we should be ok :)